src/prometheus_alert_rules/nginx/knyar-nginx-exporter.yml

```diff
@@ -0,0 +1,23 @@
+# Obtained from:
+# https://samber.github.io/awesome-prometheus-alerts/rules#nginx-1
+
+groups:
+- name: KnyarNginxExporter
```

Comment:
What does the `Knyar` prefix stand for?

Reply:
It is the name of the author: https://samber.github.io/awesome-prometheus-alerts/rules#nginx

Reply:
Sure, I can it up a bit.
Anyway keep in mind that this is the name of the `group` not the `alert`

![image](https://github.com/canonical/mimir-coordinator-k8s-operator/assets/939888/228bc958-6c63-4256-a6f2-949406a4445d)

Reply:
Right! Never mind then :)